tests/opf/test_solver.py

```diff
@@ -73,12 +70,12 @@ def test_bad_gen_bus(self):
             solve_opf(self.case, opftype="AC")
 
 
-class TestAPICase9(unittest.TestCase):
-    # Test configurations of case9 with known optimal values. We should
+class TestAPICase5_PJM(unittest.TestCase):
```

Comment:
As above, we should not replace the tests, but add new ones. If the old tests fail, we should know why so we can document it.